packages/mobile/src/hooks/useDrawer.ts

```diff
@@ -14,11 +16,32 @@ import { Drawer, setVisibility } from 'app/store/drawers/slice'
  */
 export const useDrawer = (
```

Comment:
So it looks like `useDrawer` is only used for NowPlaying right?

Can we maybe rename `useNativeDrawer` since that will be the default once we aren't dependent on web?

Reply:
yeah exactly, and sure can def update